src/utils/colonyActions.ts

```diff
@@ -27,3 +33,25 @@ export const getDetailsForAction = (
     };
   }, {});
 };
+
+/*
+ * Get values for action type based on action type
+ */
+export const getValuesForActionType = (
+  args: string,
+  actionType: ColonyActions,
+): ValuesForActionTypesMap => {
+  const argsObj = JSON.parse(args);
+  switch (actionType) {
+    case ColonyActions.MintTokens: {
+      return {
+        initiator: argsObj.agent,
+        recipient: argsObj.who,
+        amount: argsObj.amount,
+      };
+    }
+    default: {
+      return {};
+    }
+  }
+};
```

Comment:
I like this idea!

However:
- this should also handle the other actions which are not coming from events directly (eg: OneTxPayment)
- this should use a static map same as we do with the other utils

Reply:
For the first dot -> im disagree, i would keep this only for events, when we parsing "args" object / String
For other edge cases like OneTxPayment we should think about something scalable, but it will be easier  to create it, when we will have at least second case like OneTxPayment. Not here. 
For the second dot - i will create static map

Reply:
> For the first dot -> im disagree, i would keep this only for events, when we parsing "args" object / String

In that case find a better name for that utis, as it's current name implies that it gets values for actions, including ones like OneTx, which will not be the case here.

I don't know, maybe something like `getValuesForSingleEventAction` ?